interactions:
- request:
    body: |
      {"data":{"attributes":{"account_tags":["key:value"],"auth_config":{"role_name":"DatadogIntegrationRole"},"aws_account_id":"123456789012","aws_partition":"aws","logs_config":{"lambda_forwarder":{"lambdas":["arn:aws:lambda:us-east-1:123456789012:function:DatadogLambdaLogForwarder"],"log_source_config":{"tag_filters":[{"source":"s3","tags":["test:test"]}]},"sources":["s3"]}},"metrics_config":{"automute_enabled":true,"collect_cloudwatch_alarms":true,"collect_custom_metrics":true,"enabled":true,"tag_filters":[{"namespace":"AWS/EC2","tags":["key:value"]}]},"resources_config":{"cloud_security_posture_management_collection":false,"extended_collection":false},"traces_config":{}},"type":"account"}}
    form: {}
    headers:
      Accept:
      - application/json
      Content-Type:
      - application/json
    id: 0
    method: PATCH
    url: https://api.datadoghq.com/api/v2/integration/aws/accounts/448169a8-251c-4344-abee-1c4edef39f7a
  response:
    body: '{"errors":[{"status":"404","title":"Account not found","detail":"AWS account
      with provided id is not integrated"}]}'
    code: 404
    duration: 0ms
    headers:
      Content-Type:
      - application/vnd.api+json
    status: 404 Not Found
version: 2
